tests/acceptance/features/apiOcisSpecific/apiWebdavPreviews-previews.feature

```diff
@@ -1,112 +0,0 @@
-@api @issue-ocis-187
-Feature: previews of files downloaded through the webdav API
```

Comment:
Why did these get deleted?
I don't see where previews was fixed in core - `expected-failures.txt` has a whole list of core scenarios `apiWebdavPreviews/previews.feature:*` that are still expected failures.

In a separate PR I can try what happens if these "bug demo scenarios" are put back.